packages/csharp/PortingAssistant/PortingAssistant.Telemetry/PortingAssistant.Telemetry.csproj

```diff
@@ -12,5 +12,6 @@
     <PackageReference Include="PortingAssistant.Client.Client" Version="1.8.41-g07201d3178" />
     <PackageReference Include="System.IO.FileSystem" Version="4.3.0" />
     <PackageReference Include="PortingAssistant.Client.Telemetry" Version="1.8.41-g07201d3178" />
+    <PackageReference Include="System.Web.Helpers.Crypto" Version="3.2.3" />
```

Comment:
Future work log: This will conflict with main branch. No need to merge this file. Required codes are already in the main branch.